<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>datetimewindow</class>
 <widget class="QWidget" name="datetimewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/tu/Res/btn_nosel.png);</string>
  </property>
  <widget class="QPushButton" name="m_pBtn_Year_Up">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_up.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLb_Year">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;font-size: 50px;font: bold;background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Year_Down">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_down.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>67</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/spliter.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Month_Up">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_up.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Day_Up">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>170</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_up.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Hour_Up">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>170</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_up.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Min_Up">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>170</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_up.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>160</y>
     <width>67</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/spliter.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_pBtn_Month">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>320</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;font-size: 50px;font: bold;background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_pBtn_Day">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>320</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;font-size: 50px;font: bold;background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Month_Down">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>450</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_down.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Day_Down">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>450</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_down.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Hour_Down">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>450</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_down.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Min_Down">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>450</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_down.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_pBtn_Hour">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>320</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;font-size: 50px;font: bold;background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_pBtn_Min">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>320</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;font-size: 50px;font: bold;background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Save">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>640</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/save.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Return">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>720</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/return.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
